src/report_formats/XML/report_format.xml

```diff
@@ -2,7 +2,7 @@
   <name>XML</name>
   <summary>Raw XML report.</summary>
   <description>
-    Complete scan report in GMP XML format.
+    Anonymized scan report in GMP XML format.
```

Comment:
This should have been changed in the Anonymous_XML/report_format.xml instead. Opened a PR #1191 to fix this.

Reply:
thanks!